client/src/session.rs

```diff
@@ -1721,10 +1733,20 @@ mod tests {
 
         // Function should finish in timeout and return error.
         assert!(
-            timeout(Duration::from_millis(4500), layer1.layer.session(layer2.id))
+            timeout(os_specific_timeout(), layer1.layer.session(layer2.id))
                 .await
                 .unwrap()
                 .is_err()
         );
     }
+
+    #[cfg(not(target_family = "windows"))]
+    fn os_specific_timeout() -> Duration {
+        Duration::from_millis(4500)
+    }
+
+    #[cfg(target_family = "windows")]
+    fn os_specific_timeout() -> Duration {
+        Duration::from_millis(6000)
+    }
```

Comment:
Does it make sense to optimize non-windows timeout with 1.5s ? I guess 6s for every target should be fine*.

*Not to mention that such long tests shouldn't be made on unit level at first place.

Reply:
🤷🏻‍♂️ As I commented above, test failed few times on Windows@CI. I could reproduce it on my local Windows host. Increasing timeout fixed it. Unfortunately we will not get self hosted Windows machine, but still IMO such tests are like a tripwire indicating that something changed.

Reply:
I think it is not worth having two functions depending on multiple targets family.

Reply:
It has been replaced with same value for both OS.